docs/release/trg-0/trg-1-4.md

```diff
@@ -0,0 +1,71 @@
+---
+title: TRG 1.04 - Diagrams as Code
+---
+
+| Status | Created       | Post-History          |
+|--------|---------------|-----------------------|
+| Draft  | 20-Sept-2023  | created initial Draft |
+|        |               |                       |
+
+## Why
+
+To have an easier approach to update current static files and have a better overview of the changes within the product pictures.
+
+This helps people to understand our workflows within our products and make it easier to understand the product itself.
+Also updating the static files will be easier and faster to do for every committer or maintainer of our repositories.
+
+## Description
+
+All static files **must** be created with code and also stored as `.svg` file within your product repository.
+After a committers [decision](https://github.com/eclipse-tractusx/sig-infra/discussions/19) there are 2 languages accepted, either [PUML](https://plantuml.com/en/) or [Mermaid](https://mermaid.js.org/).
+These code files (`.puml`,`.mmd`,`mermaid`) and static `.svg` files **must** be stored within the `docs/static` folder.
```

Comment:
It's possible to include mermaid directly into the markdown files in GitHub.

For doing this you simply put the code into a code block and specify mermaid as the language:
```plain
```mermaid
flowchart TD
  ...
```

Shouldn't this bee a solution as well?

Reply:
https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/401/commits/b8f9f8ff878110ec1781b2f56eafb0474db42d58#diff-400d046ef5dbe121eec51e7153a73522502188ff428733d20ec32f64b3ae3c76R22